charts/oes/templates/oes-sapor-deployment.yaml

```diff
@@ -0,0 +1,52 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    app: oes
+    component: sapor
+  name: sapor
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: oes
+      component: sapor
+  template:
+    metadata:
+      annotations:
+        configmap/checksum: {{ (.Files.Glob "config/*").AsConfig | sha256sum }}
+      labels:
+        app: oes
+        component: sapor
+    spec:
+      containers:
+      - image: {{ .Values.oes.image.sapor.repository }}:{{ .Values.oes.image.sapor.tag }}
+        name: sapor
+        ports:
+        - containerPort: 8085
+          protocol: TCP
+        volumeMounts:
+        - name: api-volume
+          mountPath: /opt/opsmx/
+        - name: certs-volume
+          mountPath: /etc/pki/ca-trust/extracted/java
+        readinessProbe:
+          tcpSocket:
+            port: 8085
+          initialDelaySeconds: 120
+          periodSeconds: 30
+        livenessProbe:
+          httpGet:
+            path: /actuator/health
+            port: 8085
+          initialDelaySeconds: 180
+          periodSeconds: 60
+      imagePullSecrets:
+      - name: regcred
+      volumes:
+        - name: api-volume
+          configMap:
+            name: sapor-config
+        - name: certs-volume
```

Comment:
Why is this mandatory and not optionally overwritten by the end user if they have custom cacerts? Why are we not bundling the default cacerts that java uses?